### PR TITLE
feat: Added TextArea component

### LINKED ISSUE #762: Implement style guide in code
We need to implement the components specified in the [style guide](https://www.figma.com/file/Wrn8Zg87HKtNPNDX5gqMKs/HM-3.0-Style-Guide) so that we can move forward with creating pages.

## Form Components

- [x] Buttons
- [x] Text fields
- [x] Dropdowns
- [x] Checkboxes
- [x] Radio buttons
- [x] Text area

## Layout Components

- [x] Cards
- [x] Vertical groups
- [x] Horizontal groups (navbar)
- [x] Tables

### PR DESCRIPTION
**Pull request by skyegallup**
Closes #762 (hopefully!)

image

# Contents

- Added TextArea component

### CODE REVIEW

File: frontend/components/base/TextArea.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<template>
```
[the remark this answers is not in the document]
Comment (skyegallup) [reply frontend/components/base/TextArea.vue:1]: ok and?

File: frontend/components/base/TextArea.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+</template>
+
+<script>
+export default {
+    name: 'TextArea',
+    props: {
+        placeholder: String,
+        cols: {
+            type: Number,
+            default: 3
```
Comment (peterkos) [inline frontend/components/base/TextArea.vue]: nit: should we do default 20 instead? per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#attr-cols

**Review by peterkos — APPROVED** [review]
one nit, LGTM!

File: frontend/components/base/TextArea.vue (hunk 2 above)
Comment (skyegallup) [reply frontend/components/base/TextArea.vue]: done :+1:

**Pull request merged**